web/styles/zulip.css

```diff
@@ -858,38 +857,38 @@ div.focused-message-list.is-conversation-view .recipient_row {
     }
 }
 
+#topic_edit_form {
+    display: flex;
+    align-items: center;
+    gap: 3px;
+    margin: 0;
+    padding-right: 5px;
+}
+
 .inline_topic_edit {
-    margin-bottom: 5px;
-    width: 206px;
+    flex: 1;
+    /* Set the max width of the inline topic edit input to the realm's
+       max_topic_length (60 chars) + 5 chars extra for some cushion. */
+    max-width: 65ch;
+    line-height: 1.2142em;
```

Comment:
This seems like a very specific number. How was it arrived at?

Reply:
I have used the same `line-height` as that used for the stream topic:
```css
.message_header.message_header_stream .stream_topic {
    /* ... */
    line-height: 1.2142em;
    /* ... */
}
```

Reply:
The logic behind this being that there should be minimal visual change when switching to the inline topic edit UI, giving it the feel that one is actually editing the text which was previously visible.